01_Inleiding.Rmd

```diff
@@ -65,7 +67,8 @@ kable(gw_types_groupen,
 ```
     
 
-_GT-groep 5 bevat uitsluitend grondwaterafhankelijke HT die zodanig breed gedefinieerd zijn dat ze op sommige locaties grondwatergevoed zijn, maar soms tot meestalniet._ **_De vraag is of we deze groep al dan niet meenemen._**  
+_GT-groep 5 bundelt grondwaterafhankelijke HT die zodanig breed gedefinieerd zijn dat ze op sommige locaties grondwatergevoed zijn, maar soms tot meestal niet.
+_ **_De vraag is of we deze groep meenemen._**  
```

Comment:
Best meteen hierna een zin plaatsen die aangeeft wat het voordeel (+ nadeel?) is van groep 5. Voordeel is alvast snelle reactie op droogte. Bij nieuwe locaties selecteren (nieuwe peilbuizen) is de voorwaarde alvast dat een locatie grondwatergevoed moet zijn (shallow groundwater), dus in groep 5 kan dat een (beperkt?) nadeel geven.

Reply:
OK, heb dit als discussiepunt toegevoegd